runtime/stout/src/lib.rs

```diff
@@ -345,6 +350,7 @@ impl pallet_collator_selection::Config for Runtime {
 impl pallet_sudo::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = ();
```

Comment:
See related comment for trappist runtime.

Reply:
this can be included in the next release.